sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.NUnit/README.md

```diff
@@ -84,14 +87,19 @@ Run Playwright tests against browsers managed by the service using the configura
 dotnet test --logger "microsoft-playwright-testing"
 ```
 
+or run using runsettings file, check here for [sample](https://aka.ms/mpt/sample-nunit) runsettings file 
```

Comment:
Alternatively, you can configure the tests by using a runsettings file. For a sample runsettings file, refer to the [sample nunit documentation](link).